Example/Example/AttachmentDetailsViewController.swift

```diff
@@ -67,7 +70,11 @@ class AttachmentDetailsViewController: UITableViewController
             return
         }
         let alt = altTextField.text
-        onUpdate(alignment.toAttachmentAlignment(), size.toAttachmentSize(), url, alt)
+        var linkURL: URL?
+        if let linkText = linkURLTextField.text {
+            linkURL = URL(string: linkText)
```

Comment:
We could condense this like:

```
var linkURL = URL(string: linkURLTextField.text ?? "")
```

If you pass in an empty string, you get back `nil`.